packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -1,33 +1,201 @@
+  transitionCssVar: '--fui-Progress--transition',
+};
+
+const barThicknessValues = {
+  medium: '2px',
+  large: '4px',
+};
+
+const IndeterminateProgress = {
+  '0%': {
+    left: '0%',
+  },
+  '100%': {
+    left: '100%',
+  },
 };
 
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
+const useRootStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'grid',
+    ...shorthands.gap('8px', '0px'),
+    ...shorthands.overflow('hidden'),
+  },
+});
+
+/**
+ * Styles for the title
+ */
+const useLabelStyles = makeStyles({
+  default: {
+    gridRowStart: '1',
+    ...typographyStyles.body1,
+    color: tokens.colorNeutralForeground1,
+  },
+});
+
+/**
+ * Styles for the description
+ */
+const useDescriptionStyles = makeStyles({
+  default: {
+    gridRowStart: '3',
+    ...typographyStyles.caption1,
+    color: tokens.colorNeutralForeground2,
+  },
+});
+
+/**
+ * Styles for the progress bar
+ */
+const useBarStyles = makeStyles({
+  base: {
+    gridColumnStart: '1',
+    gridRowStart: '2',
+    backgroundColor: tokens.colorCompoundBrandBackground,
+
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: 'Highlight',
+    },
+  },
+  medium: {
+    height: barThicknessValues.medium,
+  },
+  large: {
+    height: barThicknessValues.large,
+  },
+  determinate: {
+    width: `var(${progressCssVars.percentageCssVar})`,
+  },
+  nonZeroDeterminate: {
+    width: `var(${progressCssVars.percentageCssVar})`,
+    transitionProperty: 'width',
+    transitionDuration: '0.3s',
+    transitionTimingFunction: 'ease',
+  },
+  indeterminate: {
+    maxWidth: '33%',
+  },
+  ltr: {
+    position: 'relative',
+    backgroundImage: `linear-gradient(
+      to right,
+      ${tokens.colorNeutralBackground6} 0%,
+      ${tokens.colorCompoundBrandBackground} 50%,
+      ${tokens.colorNeutralBackground6} 100%
+    )`,
+    animationName: IndeterminateProgress,
+    animationDuration: '3s',
+    animationIterationCount: 'infinite',
+  },
+
+  rtl: {
+    animationDirection: 'reverse',
   },
+});
+
+const useTrackStyles = makeStyles({
+  base: {
+    gridRowStart: '2',
+    gridColumnStart: '1',
+    backgroundColor: tokens.colorNeutralBackground6,
 
-  // TODO add additional classes for different states and/or slots
+    '@media screen and (forced-colors: active)': {
+      ...shorthands.borderBottom('1px', 'solid', 'CanvasText'),
+    },
+  },
+  medium: {
+    height: barThicknessValues.medium,
+  },
+  large: {
+    height: barThicknessValues.large,
+  },
 });
 
 /**
  * Apply styling to the Progress slots based on the state
  */
 export const useProgressStyles_unstable = (state: ProgressState): ProgressState => {
-  const styles = useStyles();
-  state.root.className = mergeClasses(progressClassNames.root, styles.root, state.root.className);
+  const { indeterminate = false, thickness = 'medium', percentComplete = 0 } = state;
```

Comment:
The styles function shouldn't be providing default values; that should happen in the `useProgress` function, and the state should have these props be `Required` so they aren't undefined by the time they get to `useProgressStyles`.

.. And it looks like that is all already the case, so these defaults can probably just be removed?
```suggestion
  const { indeterminate, thickness, percentComplete } = state;
```


I apologize if I previously said you should add defaults here; I might have mistaken the `useProgressStyles` function for `useStyles`.